src/meta/src/stream/scale.rs

```diff
@@ -1656,6 +1721,7 @@ impl ScaleController {
             .cloned()
             .collect_vec();
 
+        // TODO: Does SourceBackfill need this?
```

Comment:
It's a no-shuffle downstream

Reply:
Oh, I think I get it. `MergeNode` needs a special treatment here because we need to update its `upstream_actor_id`. Other nodes like `SourceBackfill` doesn't need to do this.